Common/Orders/Order.cs

```diff
@@ -285,5 +285,9 @@ public static Order CreateOrder(SubmitOrderRequest request)
             }
             return order;
         }
+        /// <summary>
+        /// Order Expiry on a specific UTC time.
+        /// </summary>
+        public DateTime DurationValue;
```

Comment:
With the three different variations of the Order constructors; because the Duration is set as OrderDuration.GTC, I'll set the duration value by default as DateTime.MaxValue as discussed.